web/wp-content/themes/alps-v3/resources/piklist/parts/widgets/author-box.php

```diff
@@ -0,0 +1,69 @@
+<?php
+  /*
+    Title: Author Box
+    Description: * This block will display a block of the post author's profile information.
+  */
+?>
+
+<div class="c-block c-block__text has-image u-theme--border-color--darker u-border--left c-block__text-expand u-spacing u-background-color--gray--light u-padding u-clear-fix can-be--dark-dark">
+
+    <?php
+        if(function_exists('get_avatar_url')) {
+            $avatar_url = get_avatar_url( get_the_author_meta( 'email' ), array("size"=>400));
+         }
+    ?>
+        <img class="c-block__image" itemprop="image" src=' <?php echo $avatar_url ?> '>
+
+    <h4 class="c-media-block__kicker c-block__kicker u-space--zero--top "><?php echo $settings['text_link_title']; ?></h4>
+    <h3 class="u-theme--color--darker u-font--primary--m vcard author" itemprop="url" rel="author">
+        <a href="<?php echo esc_url( get_author_posts_url( get_the_author_meta( 'ID' ) ) ); ?>" class="c-block__title-link u-theme--link-hover--dark fn" itemprop="name">
```

Comment:
You can remove the spaces before the opening and closing `()`

Reply:
@tpitre I'm not sure about this one. I've made all the other changes and pushed them.

Reply:
@designerbrent Oh, sorry I wasn't clear. I meant these:

![img](https://cl.ly/2f9685a6d230/download/Image%2525202018-11-05%252520at%2525204.40.46%252520PM.png)

Reply:
got it.